velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -98,6 +98,35 @@ struct WeekFunction : public InitSessionTimezone<T> {
   }
 };
 
+template <typename T>
+struct YearOfWeekSparkFunction : public InitSessionTimezone<T> {
```

Comment:
Why is it `YearOfWeekSparkFunction`? The namespace is enough to distinguish them.

Reply:
That's a good point. @PHILO-HE Would you take a look?

Reply:
@jinchengchenghh, thanks for your comment! Just fixed. If you have more comment, please kindly let me know.